web/src/app/users/UserRouter.js

```diff
@@ -56,34 +52,38 @@ function UserOnCallAssignments() {
   return <UserOnCallAssignmentList userID={userID} currentUser />
 }
 
-export default class UserRouter extends Component {
-  render() {
-    return (
-      <Switch>
-        <Route exact path='/users' component={UserList} />
-        <Route
-          exact
-          path='/users/:userID'
-          render={({ match }) => (
+export default function UserRouter() {
+  const { userID } = useSessionInfo()
+
+  return (
+    <Switch>
+      <Route exact path='/users' component={UserList} />
+      <Route
+        exact
+        path='/users/:userID'
+        render={({ match }) =>
+          match.params.userID === userID ? (
+            <Redirect to='/profile' />
```

Comment:
Let's see if we can handle this in `routes.js` -- we have all of our other redirects there

Reply:
We put all static redirects in the routes file so that all (non statefull) app routing happens in one place. It also means it doesn't have to render the tree before going back up for a redirect that we already know about. We aren't doing dynamic loading atm/anymore so it's not as noticeable from performance though.

If putting it in routes ends up being too complicated, I'd be ok keeping it in UserRouter for now, but then let's switch it inro a static from/to redirect in the router, instead of mixing it in with the details page render function.